modules/configuring-cert-based-auth-quay-cloudsql.adoc

```diff
@@ -0,0 +1,163 @@
+:_content-type: PROCEDURE
+[id="configuring-cert-based-auth-quay-cloudsql"]
+= Configuring certificate-based authentication with CloudSQL
+
+Use the following procedure to configure certificate-based authentication with CloudSQL.
+
+.Prerequisites
+
+* You have generated custom Certificate Authorities (CAs) and your SSL/TLS certificates and keys are available in `PEM` format that will be used to generate an SSL connection with your CloudSQL database. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_quay/{producty}/html-single/securing_red_hat_quay/index#ssl-tls-quay-overview[SSL and TLS for {productname}].
+* You have `base64 decoded` the original config bundle into a `config.yaml` file. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_quay/{producty}/html-single/deploying_the_red_hat_quay_operator_on_openshift_container_platform/index#operator-config-cli-download[Downloading the existing configuration].
+* You are using an externally managed PostgreSQL or CloudSQL database. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_quay/{producty}/html-single/deploying_the_red_hat_quay_operator_on_openshift_container_platform/index#operator-unmanaged-postgres[Using and existing PostgreSQL database] with the `DB_URI` variable set.
+* The `postgres` component of your `QuayRegistry` CRD is set to `managed: false`, and your CloudSQL database is set with the `DB_URI` configuration variable. The following procedure uses `postgresql://<cloudsql_username>:<dbpassword>@34.29.130.58:5432/quay14`.
+* For CloudSQL, *Manage SSL mode* must be set to *Require trusted client certificates*. This can be set on the *Connections* page of your CloudSQL instance.
+
+.Procedure
+
+. After you have generated the CAs and SSL/TLS certificates and keys for your CloudSQL database and ensured that they are in `.pem` format, test the SSL connection to your CloudSQL server:
+
+.. Initiate a connection to your CloudSQL server by entering the following command:
++
+[source,terminal]
+----
+$ psql "sslmode=verify-ca sslrootcert=<ssl_server_certificate_authority>.pem sslcert=<ssl_client_certificate>.pem sslkey=<ssl_client_key>.pem hostaddr=<34.29.130.58> port=<5432> user=<cloudsql_username> dbname=<cloudsql_database_name>"
+----
+
+.. Create a new database and verify that the connection is using SSL by running the following command:
```

Comment:
this part is optional, I think we needn't so detail, as I think this should be backend knowledge and common sense to use quay

Reply:
@SeanZhao-redhat Which part? Lines 16-76? or just line 26?

Thanks!

Reply:
Lines 16-76 are optional, I review again, line 16~25 looks good, I suggest we keep it, remove line 26~76